v2/pkg/protocols/headless/engine/rules.go

```diff
@@ -8,23 +8,24 @@ import (
 
 // routingRuleHandler handles proxy rule for actions related to request/response modification
 func (p *Page) routingRuleHandler(ctx *rod.Hijack) {
-	// usually browsers don't use chunked transfer encoding so we set the content-length nevertheless
+	// usually browsers don't use chunked transfer encoding, so we set the content-length nevertheless
 	ctx.Request.Req().ContentLength = int64(len(ctx.Request.Body()))
 
 	for _, rule := range p.rules {
 		if rule.Part != "request" {
 			continue
 		}
 
-		if rule.Action == ActionSetMethod {
+		switch {
```

Comment:
I think we might even use:
```go
switch rule.Action {
    case ActionSetMethod:
    ...
}
```

Reply:
Very good point! I've transformed them automatically using the IDE, and haven't payed enough attention that it could be done better